Web3privacynowplatform/Briefs/Waku.md

```diff
@@ -0,0 +1,88 @@
+**Basic info**
+
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Project name**  | Waku |
+| **Logotype**  | https://github.com/waku-org/docs.waku.org/blob/develop/static/img/logo_white.svg |
+| **Short project description (1 sentence)** | A family of robust, censorship-resistant communication protocols designed to enable privacy-focused messaging for web3 apps. |
+| **Launch date** | March 2022  |
+| **Is it open source?** | Yes |
+| **Website**  | https://waku.org |
+| **Github**  | https://github.com/waku-org |
+
+**Token**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Do you have a token** | No |
+
+**Socials**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Blog** | https://blog.waku.org/ |
+| **Twitter (X)** | https://x.com/waku_org/waku_org |
+| **Telegram** | https://t.me/waku_org |
+| **Discord** | https://discord.waku.org |
+| **Other** | https://logos.co |
+
+**Docs**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Docs** | https://docs.waku.org |
+
+**Team**
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Is your team anon or public**  | public | 
+| **List core contributors** | https://github.com/orgs/waku-org/people |
+
+**Funding** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **How are you funded**  | mixed | 
+| **List VCs backing you**  | Logos Collective |
+| **List VCs round**  | if aplicable: pre-seed, seed, Series A, B,C | 
+| **Raised money** | add a Crunchbase link | 
+
+**Github** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **What licence is in use**  | add link to Github repo | 
+
+**Technology** -- NEEDS REVIEW
+| Request  | Data point | 
+| ------------- | ------------- |
+| **Technical specialisation**  | specify: ZK, account abstraction, mixnet etc | 
```

Comment:
off-chain messaging infra ?

Reply:
That is a valid entry IMHO

I would also add 'relay network' or 'gossip network' (whichever you think is more suited), 'spam prevention', 'DoS-mitigation', 'pseudonym supported', 'modular design', 'peer-to-peer', 'platform agnostic'

I pulled most of these words from the Waku docs: https://docs.waku.org/